types/webdriverio/index.d.ts

```diff
@@ -769,8 +769,8 @@ declare namespace WebdriverIO {
         deleteCookie(name?: string): Client<RawResult<null>> & RawResult<null>;
         deleteCookie<P>(name?: string): Client<P>;
 
-        getCookie(): Client<Cookie[]> & Cookie[];
-        getCookie(name: string): Client<Cookie> & Cookie;
+        getCookie(): Client<Cookie & RawResult<string>[]> & Cookie[] & RawResult<string>[];
```

Comment:
Travis complains about `ERROR: 772:38   array-type  Array type using 'T[]' is forbidden for non-simple types. Use 'Array<T>' instead.`. This applies to multiple lines. Can you please fix these issues?